### PR TITLE
Add allocations to template renderer subscription

### PR DESCRIPTION
**Pull request by eileencodes**
This PR adds the allocations to the instrumentation for template and
partial rendering.

Before:

```
  Rendering posts/new.html.erb within layouts/application
  Rendered posts/_form.html.erb (9.7ms)
  Rendered posts/new.html.erb within layouts/application (10.9ms)
Completed 200 OK in 902ms (Views: 890.8ms | ActiveRecord: 0.8ms)
```

After:

```
  Rendering posts/new.html.erb within layouts/application
  Rendered posts/_form.html.erb (Duration: 7.1ms | Allocations: 6004)
  Rendered posts/new.html.erb within layouts/application (Duration: 8.3ms | Allocations: 6654)
Completed 200 OK in 858ms (Views: 848.4ms | ActiveRecord: 0.4ms | Allocations: 1539564)
```

cc/ @tenderlove

### CODE REVIEW

File: actionpack/lib/action_controller/log_subscriber.rb [hunk 1]
```diff
@@ -20,12 +20,15 @@ def process_action(event)
       info do
         payload   = event.payload
         additions = ActionController::Base.log_process_action(payload)
-
         status = payload[:status]
```
Comment (albertoalmagro) [inline actionpack/lib/action_controller/log_subscriber.rb]: Nitpick: After removing the blank line I would format `status` to the same level as`additions` as we seem to follow that style across this file.
```ruby
payload   = event.payload
additions = ActionController::Base.log_process_action(payload)
status    = payload[:status]
```

**Review by albertoalmagro — COMMENTED** [review]
Wow thanks @eileencodes! I just added a nitpicky comment 🔍

File: actionpack/lib/action_controller/log_subscriber.rb (hunk 1 above)
Comment (eileencodes) [reply actionpack/lib/action_controller/log_subscriber.rb]: I went the other way. Evening the space makes searching harder so I'm not a huge fan.

Comment (albertoalmagro) [reply actionpack/lib/action_controller/log_subscriber.rb]: Great! Thanks for caring about this and for the PR ❤️

**Comment by nateberkopec** [comment]
Since this uses GC.stat, do we care that this will throw out potentially nonsense numbers for multithreaded application servers like Puma?

[Also, it looks like GC moving now increments the object counter, also making this measurement less accurate.](https://github.com/ruby/ruby/blob/2d6b92c5f6b6343f34bb363604339f4bd7b951ac/gc.c#L7432)

**Comment by nateberkopec** [comment]
@tenderlove @eileencodes forgot to tag.

**Pull request merged**